src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -9602,7 +9610,7 @@ void visitInterpolatedStringHandlerConstructor()
                 {
                     foreach (var placeholder in handlerData.ArgumentPlaceholders)
                     {
-                        if (placeholder.ArgumentIndex < previousArgumentConversionResults.Count && placeholder.ArgumentIndex >= 0)
+                        if (placeholder.ArgumentIndex < previousArgumentConversionResults.Count && placeholder.ArgumentIndex is >= 0 or BoundInterpolatedStringArgumentPlaceholder.ExtensionReceiver)
```

Comment:
Why do we need to remove `ExtensionReceiver` placeholder but not other special placeholders like `InstanceParameter`?

Reply:
No placeholder replacements are added for those.